source/expressions.tex

```diff
@@ -1505,7 +1511,7 @@
 \end{bnf}
 
 \pnum
-\indextext{name!component}%
+\indextext{component name}%
 The component names of a \grammarterm{qualified-id} are those of
```

Comment:
Deferred.

Reply:
The intent was that the component names of a non-terminal are like the type of an expression: a single concept that, depending on the structure of the subject, is defined in various ways.  We can, if it would clarify that situation, introduce the term itself somewhere prior to [expr.prim.id.unqual] and not italicize it there.